app/services/katello/event_queue.rb

```diff
@@ -1,15 +1,30 @@
 module Katello
   class EventQueue
+    MAX_AGE = 6.hours
```

Comment:
To make sure i follow correctly, how we prevent things from being rescheduled past the MAX_AGE, it looks like if an event is to be retried:

1. we update the event's process_after in the future by 'retry_seconds'
2. we then delete any event who was created at more than 6 hours ago

Assuming this is correct, i think this is fine.  I think we should probably log something if MAX_AGE has expired though for traceability.

Reply:
Yes, but there's a problem with that now that I thought through it some more: I envisioned a scenario where the task queue gets backed up, and for some reason the Sat server is stopped for some time > MAX_AGE. When it comes back up - *poof* all the events get deleted because they are too old after first event gets run. All because I tied MAX_AGE to `clear_events`.

I'm pushing an update to make the `reschedule_event` method itself aware of MAX_AGE, and if it doesn't get rescheduled it'll get cleared out as usual in PollerThread. Will add the logging too

Reply:
Should be good now :+1: